my-workings/domain-models.md

```diff
@@ -1,5 +1,11 @@
 # Domain Models and User Stories
 
+```
+UPDATES
+- addPlane was renamed to land
```

Comment:
Still wasn't sure about these final names. add and remove seem better in hindsight, but I was getting tripped up when switching between using land/take-off and add/remove